rdkcentral/Dobby
# RDKTV-14242: Drop ping log message severity

### Description
Reduce the Ping() log message from INFO level to DEBUG level.

This means we can run the Dobby daemon at INFO level in VBN builds without spamming the log with Ping() messages. Ping() messages can still be seen when running with debug level logging

### Test Procedure
Run DobbyDaemon with INFO level logging and observe the Ping() messages are no longer present.

Run `DobbyTool set-log-level debug`. Observe Ping() messages.

### Type of Change

### Requires Bitbake Recipe changes?
